studio/auth.py

```diff
@@ -105,16 +119,15 @@ def refresh_token(self, email, refresh_token):
         self.user['email'] = email
         self.expired = False
 
-        if time.time() - os.path.getmtime(api_key) > HOUR:
-            # Rename to ensure atomic writes to json file
-            # (technically more safe, but slower)
-            tmp_api_key = '/tmp/api_key_%s' % rand_string(32)
-            with open(tmp_api_key, 'wb') as f:
-                json.dump(self.user, f)
-                f.flush()
-                os.fsync(f.fileno())
-                f.close()
-            os.rename(tmp_api_key, api_key)
+        # Rename to ensure atomic writes to json file
+        # (technically more safe, but slower)
+        tmp_api_key = '/tmp/api_key_%s' % rand_string(32)
```

Comment:
this one should be rather os.path.join(tempfile.gettempdir(), "api_key_%s" % rand_string(32)) - otherwise, won't work on mac